integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -191,7 +196,15 @@ case class CarbonLoadDataCommand(databaseNameOp: Option[String],
           throw ex
         }
     }
-    Seq.empty
+    if(loadResultForReturn!=null && loadResultForReturn.getLoadName!=null) {
```

Comment:
@QiangCai, @ydvpankaj99  : why our checkstyle is not catching this format issues ?